src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+import java.io.*;
+import java.util.*;
+import java.util.Date;
 public class Duke {
+    static String line = "    ____________________________________________________________";
+    public static LinkedList<Task> storage = new LinkedList<Task>(); 
     public static void main(String[] args) {
+        
+        Scanner sc = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(line);
+
+        int count = 0;
+        int taskaddnum = 0;
+        while(true){
+            String inputstring = sc.nextLine();
```

Comment:
Consider inputString instead, to follow coding standards of camelCase variables.